lib/sdocred.c

```diff
@@ -350,61 +336,61 @@ sdo_ownership_voucher_t *sdo_ov_hdr_read(sdor_t *sdor, sdo_hash_t **hmac,
 		return NULL;
 	}
 
-	/*
-	if (!sdo_begin_readHMAC(sdor, &sig_block_start))
+	if (!sdor_array_length(sdor, &num_ov_items) || num_ov_items > 6) {
```

Comment:
Add a comment on what should the expected array should be.

Reply:
Done.